### ISSUE TITLE
Error message is unclear when wrongly importing module

### ISSUE THREAD

**Issue by Nautigsam** [denoland/deno#2338]
I have the two following modules:

```javascript
// external.ts
export default external = "external";

// fail.ts
import external from "external.ts";
console.log(external);
```
Then:
```
$ deno run fail.ts
Compiling file:///home/user/fail.ts
relative URL with a cannot-be-a-base base
```
This error message is very unclear. It should at least mention the file and the line number of the import.

**Comment by athyuttamre** [comment]
Hello! I would like to work on this if possible, with some guidance. Where should I look?

**Comment by jpike88** [comment]
I just installed Deno, and run straight into this issue. This should be higher priority... no idea where to begin fixing this in my codebase.

**Comment by bartlomieju** [comment]
This issue will be fixed in #2572

**Comment by bartlomieju** [comment]
Fixed in #2572 
```
$ deno_dev fail.ts
error: Uncaught ImportPrefixMissing: relative import path not prefixed with / or ./ or ../
► js/errors.ts:33:12
```

CC @ry
